x/profiles/types/models_chain_links.go

```diff
@@ -202,6 +204,51 @@ func (b Base58Address) VerifyPubKey(key cryptotypes.PubKey) (bool, error) {
 
 // --------------------------------------------------------------------------------------------------------------------
 
+var _ AddressData = &EthAddress{}
+
+// NewEthAddress returns a new EthAddress instance
+func NewEthAddress(value, prefix string) *EthAddress {
+	return &EthAddress{Value: value, Prefix: prefix}
+}
+
+func (e EthAddress) Validate() error {
+	if len(strings.TrimSpace(e.Value)) <= len(e.Prefix) {
```

Comment:
```suggestion
	if len(strings.TrimSpace(e.Value)) <= len(strings.TrimSpace(e.Prefix)) {
```
maybe better to trim the spaces from prefix too

Reply:
I think `strings.TrimSpace` should be remove here since it is for avoiding panic in the next codeline 
`prefix, addrWithoutPrefix := e.Value[:len(e.Prefix)], e.Value[len(e.Prefix):]`.